facade/src/main/java/org/jboss/pnc/facade/rsql/RSQLProducerImpl.java

```diff
@@ -145,4 +146,9 @@ public Path apply(From<?, DB> from, RSQLSelectorPath selector) {
         };
     }
 
+    @Override
+    public <DTO> Comparator<DTO> getComparator(String rsql) {
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
```

Comment:
Is there a JIRA to implement it in the future?

Reply:
yup